### PR TITLE
refactor logic to check space creation

### PR DESCRIPTION
**Pull request by xcoulon**
minor refactoring to improve readability of the code

Signed-off-by: Xavier Coulon <xcoulon@redhat.com>

### CODE REVIEW

File: controllers/usersignup/usersignup_controller.go [hunk 1]
```diff
@@ -346,7 +346,9 @@ func (r *Reconciler) checkIfMurAlreadyExists(reqLogger logr.Logger, config toolc
 		if shouldManageSpace(userSignup) {
 			space, created, err := r.ensureSpace(reqLogger, userSignup, mur, spaceTier)
 			// if there was an error or the space was created then return to complete the reconcile, another reconcile will occur when space is created since this controller watches spaces
-			if err != nil || created {
+			if created {
+				return true, nil
+			} else if err != nil {
```
Comment (rajivnathan) [inline controllers/usersignup/usersignup_controller.go:351]: I think we should check if there was an error first and then check if created.

Comment (xcoulon) [reply controllers/usersignup/usersignup_controller.go]: Good point! done in f19b8c1

**Review by rajivnathan — APPROVED** [review]

**Review by MatousJobanek — APPROVED** [review]

**Review by ranakan19 — APPROVED** [review]
LGTM

**Pull request merged**